pkg/services/rpcsrv/client_test.go

```diff
@@ -2064,7 +2064,7 @@ func testSubClientWaitWithMissedEvent(t *testing.T, local bool) {
 		rpcSrv.subsLock.Lock()
 		defer rpcSrv.subsLock.Unlock()
 		return len(rpcSrv.subscribers) == 1
-	}, time.Second, 100*time.Millisecond)
+	}, 2*time.Second, 100*time.Millisecond)
```

Comment:
The fix with a separate error channel was valid, we need to keep it.

Reply:
Just don't make the error channel buffered. It should be a simple non-buffered channel.